### PR TITLE
rpm: add conflicts to appsupport-system-privileged.

### PR DESCRIPTION
**Pull request by krnlyng**
[rpm] Add conflicts to appsupport-system-privileged. JB#55360

### CODE REVIEW

File: rpm/droid-config-xqau51.spec [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 %define pixel_ratio 1.75
 
+Conflicts: appsupport-system-privileged
```
Comment (Thaodan) [inline rpm/droid-config-xqau51.spec:8]: Please add this to `droid-config-common.inc`.

**Review by Thaodan — CHANGES_REQUESTED** [review]

File: rpm/droid-config-xqau51.spec (hunk 1 above)
Comment (krnlyng) [reply rpm/droid-config-xqau51.spec:8]: It's device specific and cannot be in common.

**Review by mkosola — APPROVED** [review]

File: rpm/droid-config-xqau51.spec (hunk 1 above)
Comment (krnlyng) [reply rpm/droid-config-xqau51.spec]: Ah you mean the local one, i've put it there now.

**Review by Thaodan — COMMENTED** [review]
Sounds better it might also make sense to add something, like "conflict with privileged since we support unprivileged aas".

**Pull request merged**